modules/odc-customizing-user-perspective-using-CLI-and-UI.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+[id="odc-customizing-user-perspective_{context}"]
+= Customizing user perspective 
+
+The OCP web console provides two perspectives, namely *Administrator* and *Developer*. As a cluster administrator, you can show or hide a perspective to all users or a specific user role. Customizing the perspective ensures that users can view only the perspective that is applicable to their role and tasks, which avoids confusion. For example, hide the *Administrator* perspective from unprivileged users so that they cannot manage cluster resources, users, and projects or show the *Developer* perspective to users with the developer role so that they can create, deploy, and monitor applications. 
```

Comment:
```suggestion
The OCP web console provides two perspectives by default, namely *Administrator* and *Developer*. More perspectives might be available depending on installed console plugins. As a cluster administrator, you can show or hide a perspective to all users or a specific user role. Customizing the perspective ensures that users can view only the perspective that is applicable to their role and tasks, which avoids confusion. For example, hide the *Administrator* perspective from unprivileged users so that they cannot manage cluster resources, users, and projects or show the *Developer* perspective to users with the developer role so that they can create, deploy, and monitor applications. 
```